src/main/kubeconfig-manager/create-kubeconfig-manager.injectable.ts

```diff
@@ -17,6 +18,7 @@ const createKubeconfigManagerInjectable = getInjectable({
   instantiate: (di) => {
     const dependencies = {
       directoryForTemp: di.inject(directoryForTempInjectable),
+      lensProxyPortNumberState: di.inject(lensProxyPortNumberStateInjectable),
```

Comment:
haha, this is also very wordy. Wouldn't something like `lensProxyPort` by itself convey just enough information as well without being so long?

Reply:
True, renamed.